Node Blog Michael Trew

CodeDiversity opened the pull request:
@juliankohlman

CodeDiversity committed:
initial setup

CodeDiversity committed:
custom mw added

CodeDiversity committed:
gatekeeper

CodeDiversity committed:
edit

juliankohlman commented:
Michael single-word commit messages are not a habit you want to adopt. Imagine working on a team, and reading someone else's single word commit messages. You'd have to spend extra time just figuring out what the message is referring to. Like you last commit that reads `edit`. Edit what? Read [this](https://chris.beams.io/posts/git-commit/) and try and model your commits after the examples 👍

CodeDiversity committed:
updates

CodeDiversity committed:
working on get

mtrew2015 commented:
> Michael single-word commit messages are not a habit you want to adopt. Imagine working on a team, and reading someone else's single word commit messages. You'd have to spend extra time just figuring out what the message is referring to. Like you last commit that reads `edit`. Edit what? Read [this](https://chris.beams.io/posts/git-commit/) and try and model your commits after the examples 👍

10-4 thank you for the advice @juliankohlman

CodeDiversity committed:
updated catch in posts router

mtrew2015 commented on package.json:
```diff
@@ -0,0 +1,18 @@
+    "helmet": "^3.15.0",
+    "knex": "^0.15.2",
+    "morgan": "^1.9.1",
+    "nodemon": "^1.18.7",
```
Thank you @FireInjun

CodeDiversity committed:
post router is completed

CodeDiversity committed:
all user router endpoints completed

CodeDiversity committed:
fixed code and practicing using VIM for commit messages

I see the importance of writing good commit messages, as if I were to be on a team and write a one line commit message like "edit" everyone would be confused, plus it wouldn't benefit me if I had to revert to a prior change down the line.
